### PR TITLE
[Tests-Only] Splitted acceptance-test suite apiWebdavMove

### PR DESCRIPTION
**Pull request by kiranparajuli589**
## Related Issue

- Part of #36859

## How Has This Been Tested?

- :robot:

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

File: tests/acceptance/config/behat.yml [hunk 1]
```diff
@@ -306,7 +306,17 @@ default:
         - WebDavLockingContext:
         - WebDavPropertiesContext:
 
-    apiWebdavMove:
+    apiWebdavMove1:
+      paths:
+        - '%paths.base%/../features/apiWebdavMove'
```
Comment (phil-davis) [inline tests/acceptance/config/behat.yml]:
```suggestion
        - '%paths.base%/../features/apiWebdavMove1'
```

File: tests/acceptance/config/behat.yml [hunk 2]
```diff
@@ -306,7 +306,17 @@ default:
...
+      context: *common_ldap_suite_context
+      contexts:
+        - FeatureContext: *common_feature_context_params
+        - LoggingContext:
+        - OccContext:
+        - WebDavPropertiesContext:
+
+    apiWebdavMove2:
       paths:
         - '%paths.base%/../features/apiWebdavMove'
```
Comment (phil-davis) [inline tests/acceptance/config/behat.yml]:
```suggestion
        - '%paths.base%/../features/apiWebdavMove2'
```

Comment (kiranparajuli589) [reply tests/acceptance/config/behat.yml]: done!

File: tests/acceptance/config/behat.yml (hunk 1 above)
Comment (kiranparajuli589) [reply tests/acceptance/config/behat.yml]: done!

**Comment by phil-davis** [comment]
https://drone.owncloud.com/owncloud/core/23134/59/13
`apiWebdavMove1` has 55 scenarios, 23 minutes

https://drone.owncloud.com/owncloud/core/23134/60/13
`apiWebdavMove2` has 55 scenarios, 22 minutes

It is actually just a coincidence that both suites have exactly the same number of scenarios. They really are running different features.

**Review by phil-davis — APPROVED** [review]

**Pull request merged**